main/urls.py

```diff
@@ -1,7 +1,12 @@
+# general imports
 from django.urls import path
 from main.views import home
 
+# api imports
 
+# api urls
+
+# general urls
 urlpatterns = [
-    path('', home, name="home")
+    path('', home, name="home"),
```

Comment:
Trailing commas are important to avoid conflicts with branches that require commas to add more lines